articles/storage/common/storage-use-azcopy-configure.md

```diff
@@ -107,6 +107,8 @@ Express this value in gigabytes (GB).
 | **Linux** | `export AZCOPY_BUFFER_GB=<value>` |
 | **macOS** | `export AZCOPY_BUFFER_GB=<value>` |
 
+Please note that there is always an additional overhead in terms memory usage for job tracking purposes, and this amount varies based on the number of transfers in a job. Since the buffers are the biggest component of memory usage, you can control it with `AZCOPY_BUFFER_GB` to meet you requirements.
```

Comment:
your requirements.

Reply:
also, can we do the highlighted note here 
also, can we call out that there is no flag to cap total memory usage of azcopy